bootstrap.sh

```diff
@@ -1 +1,9 @@
 #!/bin/bash
+
+kubectl apply -f st-service.yml
+kubectl apply -f st-secret.yml
+kubectl apply -f st-configMap.yml
+kubectl apply -f statefulSet.yml
+kubectl apply -f secret.yml
+kubectl apply -f deployment.yml
+kubectl apply -f namespace.yml
```

Comment:
Namespaces should be deployed as the first resource, because all other resources will be created within this namespace